fathomnet/scripts/fathomnet_generate.py

```diff
@@ -274,7 +275,15 @@ def parse_args() -> Arguments:
     logging.basicConfig(level=level)
     
     # Parse list of concepts
-    concepts = args.concepts
+    if args.concepts:
+        concepts = args.concepts
+    elif args.concepts_file:
+        if os.path.isfile(args.concepts_file):
+            with open(args.concepts_file, 'r') as ff:
+                concepts = list(ff)
+                ff.close()
```

Comment:
Using `open()` with a context manager like you're doing has the benefit of closing the file automatically, so no close needed. :)